src/TLuaInterpreter.cpp

```diff
@@ -6363,27 +6364,22 @@ int TLuaInterpreter::getMudletHomeDir(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getMudletLuaDefaultPaths
 int TLuaInterpreter::getMudletLuaDefaultPaths(lua_State* L)
 {
-    int index = 1;
+    int index = 0;
+    // This is a static method so does not have a this pointer - so go and get
+    // the one used by the profile so we can read the instance's data:
```

Comment:
Are you sure we need this explanation? This is a pattern we use in every single function in Lua. It's not really complex code that needs an explanation of "why": https://dev.to/andreasklinger/comments-explain-why-not-what-and-2-more-rules-on-writing-good-comments (sans leaving your name... because we have git for that)

Reply:
I guess so - it hadn't struck me before (at least, in a manner that caused it to *dig-in*! :open_mouth:) before that our core of Lua API functions are all `static` like that.  I guess it is something to do with linking to the Lua library in a `C` rather than `C++` like manner? :man_shrugging: 

I will zap it out...

Reply:
Done.